terraform-beyond-layer0/example-3/env-dev/main.tf

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
How do you feel about `example-3`? 

I know I've been going back and forth on this configuration, but having now looked at it, I feel like it's a lot of copied configuration with little value added. 

The only take-away I'm getting is how to keep terraform configurations separated from one another - something we could just tell the users in a small doc. 

The only other argument I can think for using this pattern would be if you needed different configurations for each environment - but I'd argue that can be achieved by adding variables and more interpolation to the modules (like how you do different scales for each environment in `example-2`).